talon/quotations.py

```diff
@@ -463,18 +481,39 @@ def split_emails(msg):
 
     Return the corrected markers
     """
-    delimiter = get_delimiter(msg)
-    msg_body = preprocess(msg, delimiter)
+    msg_body = _replace_link_brackets(msg)
+
     # don't process too long messages
     lines = msg_body.splitlines()[:MAX_LINES_COUNT]
-    markers = mark_message_lines(lines)
+    markers = mark_message_lines(lines, ignore_initial_spaces=True)
+
+    markers = _mark_quoted_email_splitlines(markers, lines)
 
     # we don't want splitlines in header blocks
     markers = _correct_splitlines_in_headers(markers, lines)
 
     return markers
 
 
+def _mark_quoted_email_splitlines(markers, lines):
+    """
+    When there are headers indented with '>' characters, this method will attempt to identify if the header is a
```

Comment:
Can you please split the comment lines so that each line is shorter than 80 chars (to comply with PEP-8)? I know that the library doesn't exactly follow this golden rule but we should try :)